src/Hooks/useFetchMonitorsWithChecks.js

```
@@ -0,0 +1,71 @@
+import { useMonitorUtils } from "./useMonitorUtils";
+
+export const useFetchMonitorsWithChecks = ({
+	teamId,
+	types,
+	limit,
+	page,
+	rowsPerPage,
+	filter,
+	field,
+	order,
+	triggerUpdate,
```

ESLint

[error] 'triggerUpdate' is defined but never used.

(no-unused-vars)